app/namespace_test.go

```diff
@@ -1296,3 +1306,354 @@ func (s *NamespaceTestSuite) TestDelete() {
 	}, nil).Times(1)
 	s.NoError(s.RunCmd("namespace", "delete", "--namespace", "ns1"))
 }
+
+func (s *NamespaceTestSuite) TestCreateExportSink() {
+	ns := "namespace"
+	type morphGetResp func(*namespaceservice.GetNamespaceResponse)
+	type morphCreateSinkReq func(*namespaceservice.CreateExportSinkRequest)
+
+	tests := []struct {
```

Comment:
[Nit] add test case name as argument to make it clear what the intent of each of these table tests is

Reply:
or if we are expecting an error, consider checking if expected error string is returned